operator/controllers/common/helperfunctions.go

```diff
@@ -22,3 +28,14 @@ func GetItemStatus(name string, instanceStatus []klcv1alpha2.ItemStatus) klcv1al
 func GetAppVersionName(namespace string, appName string, version string) types.NamespacedName {
 	return types.NamespacedName{Namespace: namespace, Name: appName + "-" + version}
 }
+
+func GetOldStatus(statuses []klcv1alpha2.ItemStatus, definitionName string) apicommon.KeptnState {
```

Comment:
please add a short comment explaining the difference of this function compared to `GetItemStatus`, which looks quite similar at first glance. Also, to stay consistent, I suggest to keep the ordering of the parameters the same as in that other function